src/Domain/LeanCode.DomainModels.EF/ModelBuilderExtensions.cs

```diff
@@ -71,7 +71,7 @@ public static void EnableOptimisticConcurrency<TEntity, TRowVersion>(
     )
         where TEntity : class, IOptimisticConcurrency
     {
-        builder.Entity<TEntity>().IsOptimisticConcurrent(useExplicitBackingFields, addRowVersion);
+        builder.Entity<TEntity>().IsOptimisticConcurrent<TEntity, TRowVersion>(useExplicitBackingFields, addRowVersion);
```

Comment:
But why?

Reply:
Previously it resolved to `IsOptimisticConcurrent<TEntity>` so it was always `byte[]` instead of what was passed as generic argument.